cloudsql/sqlserver/database-sql/service-account.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START cloud_sql_sqlserver_databasesql_gke_quickstart_sa]
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: <YOUR-KSA-NAME> # TODO(developer): replace this value

```

Comment:
nit: we should consider adding this TODO everywhere we use brackets or remove it.